activiti-cloud-services-org/activiti-cloud-services-org-core/src/main/java/org/activiti/cloud/organization/core/rest/client/feign/ProcessModelReferenceService.java

```diff
@@ -24,7 +24,8 @@
  */
 @FeignClient(url = "#{'${activiti.cloud.modeling.url:activiti-cloud-modeling}'}",
         name = "activiti-cloud-modeling",
-        path = "/v1/process-models")
+        path = "/v1/process-models",
+        configuration = FeignConfiguration.class)
```

Comment:
@constantin-ciobotaru I think that we can configure all the clients with a global configuration when we add the @EnableFeignClients annotation.. I would rather do it there.